### PR TITLE
[ADD] stock_by_warehouse_mrp_bom: New stock_by_warehouse module added for mrp.bom.line model t#52780

### PR DESCRIPTION
**Pull request by agarzaarvizu**

### CODE REVIEW

**Comment by agarzaarvizu** [comment]
Screenshot from 2022-01-12 18-43-03

File: stock_by_warehouse_mrp_bom/models/mrp_bom.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    warehouses_stock = fields.Text(store=False, readonly=True)
+    warehouse_id = fields.Many2one(
+        string="Warehouse",
+        related='bom_id.picking_type_id.warehouse_id')
+    warehouses_stock_recompute = fields.Boolean(store=False, readonly=False)
+
+    @api.multi
+    def _compute_get_warehouses_stock(self):
+        __import__('pdb').set_trace()
```
Comment (goliveirab) [inline stock_by_warehouse_mrp_bom/models/mrp_bom.py]: Remove pdb

File: stock_by_warehouse_mrp_bom/README.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product on the mrp_bom line to see the quantity of product
+by warehouse in order to help you to review such information as fast as possible.
+
+.. image:: http://screenshots.vauxoo.com/nhomar/stock_by_warehouse_sale_1.png
```
Comment (CarmenMiranda) [inline stock_by_warehouse_mrp_bom/README.rst]: Hi, It should be great to have the new screenshots here you can use as an example the ones added in the [`stock_by_warehouse_stock`](https://github.com/Vauxoo/addons-vauxoo/blob/12.0/stock_by_warehouse_stock/README.rst),  adding the SS that you put [in the MR comment](https://github.com/Vauxoo/addons-vauxoo/pull/1517#issuecomment-1011591319)

File: stock_by_warehouse_mrp_bom/README.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    :alt: Example on Form View
+    :width: 600px
+
+.. image:: http://screenshots.vauxoo.com/nhomar/stock_by_warehouse_sale_2.png
+    :alt: Example on Form View
+    :width: 600px
+
+Developers usage:
```
Comment (CarmenMiranda) [inline stock_by_warehouse_mrp_bom/README.rst]: We can delete this section, because this usage is for the `stock_by_warehouse` module

**Review by CarmenMiranda — CHANGES_REQUESTED** [review]
Just a few comments about the readme file, other than that LGTM 👍🏼

File: stock_by_warehouse_mrp_bom/README.rst (hunk 3 above)
Comment (CarmenMiranda) [reply stock_by_warehouse_mrp_bom/README.rst]: This section still needs to be deleted 🙈

**Review by CarmenMiranda — APPROVED** [review]
@hugho-ad LGTM 👍🏼

**Comment by hugho-ad** [comment]
@agarzaarvizu 

pls squash your commits into only one

**Comment by hugho-ad** [comment]
ps update icon image

**Comment by agarzaarvizu** [comment]
@hugho-ad commits squashed and image updated, please review.

File: stock_by_warehouse_mrp_bom/README.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product on the mrp_bom line to see the quantity of product
```
Comment (luisg123v) [inline stock_by_warehouse_mrp_bom/README.rst]:
```suggestion
Provide a nice widget in the product on the bill of material's line to see the quantity of product
```

File: stock_by_warehouse_mrp_bom/README.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+by warehouse in order to help you to review such information as fast as possible.
+
+.. image:: ./static/images/image.png
+    :alt: Example on Form View
+    :width: 600px
+
+Notes:
+------
+
+This module rely on variants and multiple warehouse management
```
Comment (luisg123v) [inline stock_by_warehouse_mrp_bom/README.rst]:
```suggestion
This module relies on variants and multiple warehouse management
```

File: stock_by_warehouse_mrp_bom/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Stock by Warehouse on MRP_BOM",
```
Comment (luisg123v) [inline stock_by_warehouse_mrp_bom/__manifest__.py]:
```suggestion
    "name": "Stock by Warehouse on MRP BoM",
```

File: stock_by_warehouse_mrp_bom/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "12.0.1.0.0",
+    "summary": """
+    Know the stock in all your warehouses with a simple click
+    from the mrp_bom line form.
```
Comment (luisg123v) [inline stock_by_warehouse_mrp_bom/__manifest__.py]:
```suggestion
    from the mrp bill of material's line form.
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by agarzaarvizu** [comment]
@hugho-ad @luisg123v Changes done, please review.

**Review by luisg123v — APPROVED** [review]

**Review by luisg123v — CHANGES_REQUESTED** [review]
Hi @agarzaarvizu,

Please reword commit to conform guidelines.

Someting like:

```
[ADD] module_name: Module to ...
# blank line between title and description
Module summary
```

**Comment by agarzaarvizu** [comment]
@luisg123v done

**Comment by luisg123v** [comment]
@agarzaarvizu not done yet:

- You still have a list instead of a title&description
- Module name is misspelled, it's mrp instead of mpr

**Comment by agarzaarvizu** [comment]
@luisg123v can you check it now? Please

**Comment by luisg123v** [comment]
@agarzaarvizu it still does not comply:
- It's still a list (excepting the 1st element)
- There's no space between title and description
- The description is not actually a description. You could paste module's summary instead

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**